test/functional/shared/cli/cli_test.go

```diff
@@ -228,7 +229,6 @@ func verifyCLIBasics(ctx context.Context, t *testing.T, test shared.RPTest) {
 	})
 
 	t.Run("Validate rad resource expose Container", func(t *testing.T) {
-		t.Skip("https://github.com/radius-project/radius/issues/3232")
```

Comment:
do you think this test is still flaky?

Reply:
Your retry logic is linear back-off with 1 second interval. I think total 10 seconds are too short to run container or app up completely in some cases. in new change, I used 5 second interval with jitter. This will give enough time to run container or app up.